app/main.py

```diff
@@ -16,4 +16,19 @@
 }
 collection_of_coins = {1, 2, 25}
 
+
```

Comment:
You need to simply pass all pre defined variables to the correct list in dict without calculation

Reply:
Thank you!